PyExpLabSys/drivers/keithley_2400.py

```diff
@@ -0,0 +1,183 @@
+""" Simple driver for Keithley 2400 SMU """
+from PyExpLabSys.drivers.scpi import SCPI
+
+
+class Keithley2400(SCPI):
+    """ Simple driver for Keithley 2400 SMU """
+
+    def __init__(self, interface, hostname='', device='',
+                 baudrate=9600, gpib_address=None):
+        if interface == 'serial':
+            SCPI.__init__(self, interface=interface, device=device,
+                          baudrate=baudrate, line_ending='\n')
+            self.comm_dev.timeout = 2
+            self.comm_dev.rtscts = False
+            self.comm_dev.xonxoff = False
+        if interface == 'lan':
+            SCPI.__init__(self, interface=interface, hostname=hostname)
+        if interface == 'gpib':
+            SCPI.__init__(self, interface=interface, gpib_address=gpib_address)
+
+    def output_state(self, output_state: bool = None):
+        """ Turn the output on or off """
+        if output_state is not None:
+            if output_state:
+                self.scpi_comm('OUTPUT:STATE 1')
+            else:
+                self.scpi_comm('OUTPUT:STATE 0')
+        actual_state_raw = self.scpi_comm('OUTPUT:STATE?')
+        actual_state = actual_state_raw[0] == '1'
+        return actual_state
+
+    def set_current_measure_range(self, current_range=None):
+        """ Set the current measurement range """
+        # TODO!
```

Comment:
You can `raise NotImplementedError` instead, to make sure that it doesn't not-do-anything silently

Reply:
Will do!